torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -17,6 +18,137 @@
 )
 
 
+@dataclass
+class ViewInfo:
+    base_index: int
+    size: Any = None
+    stride: Any = None
+    storage_offset: Any = None
```

Comment:
We can be more specific than Any?

Reply:
I had issue specially with dynamic it gets ugly, symbols and stuff like that i can try again

Reply:
Doesn't need to happen in this PR

Reply:
also addition optional complicate usages.

Reply:
done